website/docs/r/compute_instance.html.markdown

```diff
@@ -229,7 +229,8 @@ The following arguments are supported:
 	* `co-p1`: high I/O(performance-optimized) disk type.
 	* `uh-l1`: ultra-high I/O(latency-optimized) disk type.
 
-* `system_disk_size` - (Optional) The system disk size in GB, The value range is 1 to 1024. Changing this creates a new server.
+* `system_disk_size` - (Optional) The system disk size in GB, The value range is 1 to 1024. Changing this parameter will update the disk. 
+    You can extend the disk by setting this parameter to a new value, which must be between current size and the max size. Shrinking the disk is not supported.
```

Comment:
need to be more specified for the `max size`, or just not mention it.

Reply:
done